content/en/docs/concepts/security/pod-security-admission.md

```diff
@@ -30,25 +30,21 @@ The PodSecurityPolicy API is deprecated and will be
 [removed](/docs/reference/using-api/deprecation-guide/#v1-25) from Kubernetes in v1.25.
 {{< /note >}}
 
-<!-- body -->
 
-## Enabling the `PodSecurity` admission plugin
+## {{% heading "prerequisites" %}}
 
-In v1.23, the `PodSecurity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-is a Beta feature and is enabled by default.
+To use this mechanism, your cluster must enforce Pod Security admission.
 
-In v1.22, the `PodSecurity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-is an Alpha feature and must be enabled in `kube-apiserver` in order to use the built-in admission plugin.
+## Built-in Pod Security admission enforcement
 
-```shell
---feature-gates="...,PodSecurity=true"
-```
+In Kubernetes v{{< skew currentVersion >}}, the `PodSecurity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is a beta feature and is enabled by default. You must have this feature gate enabled.
+If you are running a different version of Kubernetes, consult the documentation for that release.
 
-## Alternative: installing the `PodSecurity` admission webhook {#webhook}
+### Alternative: installing the `PodSecurity` admission webhook {#webhook}
 
-For environments where the built-in `PodSecurity` admission plugin cannot be used,
-either because the cluster is older than v1.22, or the `PodSecurity` feature cannot be enabled,
-the `PodSecurity` admission logic is also available as a Beta [validating admission webhook](https://git.k8s.io/pod-security-admission/webhook).
+The `PodSecurity` admission logic is available as a [validating admission webhook](https://git.k8s.io/pod-security-admission/webhook). This implementation is also beta.
+For environments where the built-in `PodSecurity` admission plugin cannot be used (for example, because the cluster has the Pod Security feature administratively disabled), you can instead use that logic via a validating admission webhook.
```

Comment:
```suggestion
For environments where the built-in `PodSecurity` admission plugin cannot be enabled, you can instead enable that logic via a validating admission webhook.
```

- The example is a bit misleading, I'd suggest we just drop it.
- s/used/enabled/ since there are still situations where neither can be used (can't update namespace labels)

Reply:
I'm not clear on which bit you're suggesting dropping @tallclair. The overall mention of using the webhook, the context for when you might do that, or something else?

Reply:
Sorry, the bullet points were just summarizing the motivation behind the suggested change. The suggested change is all that I'm suggesting here.

Reply:
OK, I think that's resolved.